src/Data/List/Relation/Prefix/Heterogeneous/Properties.agda

```diff
@@ -54,6 +54,17 @@ module _ {a b r s e} {A : Set a} {B : Set b}
   antisym rs⇒e []       []       = []
   antisym rs⇒e (r ∷ rs) (s ∷ ss) = rs⇒e r s ∷ antisym rs⇒e rs ss
 
+module _ {a e} {A : Set a} {_≈_ : Rel A e} (≈-equiv : IsEquivalence _≈_) where
+
+  isPreorder : IsPreorder {A = List A} (Pointwise _≈_) (Prefix _≈_)
+  isPreorder = record
+    { isEquivalence = isEquivalence ≈-equiv
+    ; reflexive = fromPointwise
+    ; trans = trans (IsEquivalence.trans ≈-equiv) }
+
+  preorder : Preorder _ _ _
+  preorder = record { isPreorder = isPreorder }
+
```

Comment:
This should go into `Data/List/Relation/Prefix/Setoid` or something similar. I'm not sure how
all of this PRs rebased on top of other PRs are going to work once their dependencies get
merged tbh. I hope we won't get merge-conflicts-from-hell.

Reply:
Ah, is that where those things live. If there are going to be nasty merge conflicts I'll prepare a new PR by the time that the dependencies make it in.